### ISSUE TITLE
Investigate FlatList

### ISSUE THREAD

**Issue by sibelius** [rgovindji/react-native-atoz-list#9]
this just landed on react native core: https://github.com/facebook/react-native/commit/a3457486e39dc752799b1103ebe606224a8e8d32

we should take a look to improve performance

**Comment by pewh** [comment]
+1

**Comment by rgovindji** [comment]
Agreed I've been to busy to maintain this repo. You are welcome to submit a PR :)
